test/tbb/test_limiter_node.cpp

```diff
@@ -445,7 +450,9 @@ void test_try_put_without_successors() {
     int try_put_num{3};
     tbb::flow::buffer_node<int> bn(g);
     tbb::flow::limiter_node<int> ln(g, try_put_num);
+
     tbb::flow::make_edge(bn, ln);
+
```

Comment:
Why don't you place the `make_edge_impl` here?

Reply:
We have bug in case where limiter_node is predecessor. But in this case limiter_node is successor. In previous commit I don't sight it.

Reply:
This PR fixes the issue.
1000 launches completed successfully in my tests on CentOS 7 + GCC 7.4.

Reply:
Sounds great!